src/Payu/Response/ResponseAbstract.php

```diff
@@ -3,9 +3,13 @@
 
 abstract class ResponseAbstract
 {
-    const STATUS_APPROVED     = 200;
+    const STATUS_APPROVED = 200;
     const STATUS_UNAUTHORIZED = 401;
 
+    const LOYALITY_STATUS_SUCCESS = 200;
```

Comment:
ResponseAbstract bir soyut sınıf. Burada transaction'a bağlı bir ocnstant olamaz. Approved kelimesi de bu işlem için gayet anlamlı. Ancak INPUT_ERROR hatasını handle etmek istiyorsanız bunun için aynı standartta yeni bir constant tanımlayabilirsiniz. Hatırladığım kadarıyla PayU INPUT_ERROR hatasını payment transactionları için de dönüyor olması lazımdı. Hem puan sorgularken, hem de ödeme adımında bu hata değerlendirilebilir.